.travis.yml

```diff
@@ -1,14 +1,19 @@
+sudo: false
```

Comment:
We should refrain from using the sudo tag as it is deprecated as per https://blog.travis-ci.com/2018-11-19-required-linux-infrastructure-migration

What do we need this for?

Reply:
OK. Removed